File: Beginner/maxPower/maxPower.java
//run as java -ea maxPower
import java.util.Scanner;

public class maxPower{
	public static void main(String[] args){
		int N,count;
		String X;
		Scanner scan = new Scanner(System.in);

		N = scan.nextInt();
		assert(N>=1 && N<=100000) : "N must be 1 <= N <= 10^5";

		X = scan.next();
		count = 0;

		for(int i=N-1;i>=0;i--){
			if(X.charAt(i)=='0'){
				count++;
			}else{
				break;
			}
		}

		System.out.println(count);
	}
}

File: Beginner/maxPower/maxPower.c
#include <stdio.h>
#include <assert.h>

int main(void){
	int N,count;

	scanf("%d",&N);
	assert(N>=1 && N<=100000);

	char X[N];
	scanf("%s",X);

	count = 0;

	for(int i=N-1;i>=0;i--){
		if(X[i]=='0'){
			count++;
		}else{
			break;
		}
	}

	printf("%d\n",count);
	
	return 0;
}